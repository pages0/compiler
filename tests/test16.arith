let max =  (fun x -> (fun y -> if x >= y then x else y )) in
let a = 5 in
let b = 2 in
(max a) b