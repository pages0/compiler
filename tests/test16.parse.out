( let [ max ( fun ( x ) ( fun ( y ) ( if ( >= x y ) x y ) ) ) ] ( let [ a 5 ] ( let [ b 2 ] ( ( max a ) b ) ) ) )